src/components/tooltip/tooltip.module.scss

```
@@ -12,7 +12,16 @@
   width: max-content;
   z-index: 1;
 
-  .triangle {
+  .triangle_bottom {
+    border-color: $tooltip-background-color transparent transparent transparent;
+    border-style: solid;
+    border-width: 5px;
+    margin-top: -5px;
+    position: absolute;
+    top: 105%;
```

```suggestion
    top: calc(100% + 5px);
```
We want to be the full height, plus the 5px that is the size of the triangle itself

---

After looking into it more, I noticed there was a margin-top that I copied from triangle_left, that was adding the -5px (so it would be aligned with the middle of info icon). When I removed it top could be 100%.